docs/user_manual.md

```diff
@@ -138,6 +138,15 @@ A current best practice is to save all your job creation parameters in
 version-controlled files in your project directory. This allows you to tie
 your helios job params to changes in your application code.
 
+### Specifying an Access Token
+
+You can optionally specify an access token when creating a job by using the `--token` parameter.
+
+    $ helios create --token abc123 foo:v1 spotify/cassandra
+
+Each time the job is deployed, undeployed or removed, the token must be specified using the
```

Comment:
Good point. Just added start and stop to the list. You don't need to specify the token for inspect though.